build/Dockerfile

```diff
@@ -8,26 +18,27 @@ COPY go.mod go.mod
 COPY go.sum go.sum
 # cache deps before building and copying source so that we don't need to re-download as much
 # and so that source changes don't invalidate our downloaded layer
+# NOTE: downstream this needs to be a tarball, not a live download
 RUN go mod download
 
 # Copy the go source
 COPY . .
 
-# compile workspace controller binaries
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 GO111MODULE=on go build \
+# compile workspace controller binaries, then webhook binaries
+RUN export ARCH="$(uname -m)" && if [[ ${ARCH} == "x86_64" ]]; then export ARCH="amd64"; elif [[ ${ARCH} == "aarch64" ]]; then export ARCH="arm64"; fi && \
```

Comment:
seems it does not work correctly, go build fails with `cmd/go: unsupported GOOS/GOARCH pair linux/x86_64`
I think it would improve readability if we format in a multiline way.

Reply:
works here:

```

Reply:
are you using docker or podman to build? 

what does your OS tell you when you run `ARCH="$(uname -m)"; echo "ARCH=${ARCH}"`  ?